build-system-tests/package.json

```
@@ -0,0 +1,15 @@
+{
+  "name": "build-system-tests",
```

```suggestion
  "name": "@aws-amplify/ui-build-system-tests",
```
not that this package is every installable or cross referenced, but just for consistency 🙏